cmd/state/internal/cmdtree/shim.go

```diff
@@ -0,0 +1,43 @@
+package cmdtree
+
+import (
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runners/shim"
+)
+
+func newShimCommand(prime *primer.Values) *captain.Command {
+	runner := shim.New(prime)
+
+	params := shim.Params{}
+	cmd := captain.NewCommand(
+		"shim",
+		locale.T("shim_description"),
+		[]*captain.Flag{
+			{
+				Name:        locale.T("flag_state_shim_language"),
+				Description: locale.T("flag_state_shim_language_description"),
+				Value:       &params.Language,
+			},
+		},
```

Comment:
The ACs don't call for any flags. And flags wouldn't work for this command anyway since we want to support `--`.